package.json

```diff
@@ -100,6 +100,7 @@
     "abortcontroller-polyfill": "^1.1.9",
     "balena-auth": "^3.0.0",
     "balena-errors": "^4.0.0",
+    "balena-hup-action-utils": "1.0.0-initial-code-737718128eb1b664f5d161d182052f4400b3f344",
```

Comment:
Don't forget to point to the released version if that gets released before this PR.